tests/integration/reqs/test_subscribe.py

```diff
@@ -0,0 +1,24 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+from tests.integration.it_utils import test_async_and_sync
+from xrpl.models.requests import StreamParameter, Subscribe
+from xrpl.models.requests.unsubscribe import Unsubscribe
+
+MESSAGE_LIMIT = 3
+
+
+class TestSubscribe(IntegrationTestCase):
+    @test_async_and_sync(globals(), websockets_only=True)
+    async def test_basic_functionality(self, client):
+        req = Subscribe(streams=[StreamParameter.LEDGER])
+        await client.send(req)
```

Comment:
style nit, since you don't need to reuse `req` this is cleaner and clearer to me

```python
await client.send(Subscribe(streams=[StreamParameter.LEDGER]))
```

Reply:
we should also test different subscription types, in separate tests within this class

Reply:
I've added tests for validations, consensus, transactions, and transactions_proposed in my latest. I did not implement tests for the manifests, peer_status, and server streams. 

The manifests stream is slow because manifests don't come around all that often. The peer_status stream requires an admin connection. The server stream is also slow since the status of the server doesn't change that frequently. Let me know if the stream that I implemented are enough.